buffer/src/test/java/io/netty/buffer/ReadOnlyDirectByteBufferBufTest.java

```diff
@@ -236,6 +236,16 @@ public void testGetReadLong() {
         buf.release();
     }
 
+    @Test(expected = IndexOutOfBoundsException.class)
+    public void getDirectByteBufferBoundaryCheck() {
```

Comment:
@njhill this test even pass without your changes. So I think you need to add another test ;)

Reply:
lol ;) Thanks!